druntime/src/core/internal/vararg/aarch64.d

```diff
@@ -36,7 +36,14 @@ extern (C++, std) struct __va_list
 }
 
 ///
-alias va_list = __va_list;
+alias __va_list_tag = __va_list;
```

Comment:
If you plan on introducing that `__va_list_tag` for DMD for some bizarre reason (no occurrence in https://github.com/ARM-software/abi-aa/blob/main/aapcs64/aapcs64.rst), then for `version(DigitalMars)` only.

Reply:
The reason is Target.va_listType uses it. But I changed it to using __va_list